### PR TITLE
Add temporary rake task to remove legacy taxon links

### PR DESCRIPTION
**Pull request by BeckaL**
Legacy taxon links are now deprecated, and we want to remove the last traces of them before removing them from the taxon schema.

This rake task takes a list of the ~170 content ids of taxons that were mapped to specialist topics as per https://docs.google.com/spreadsheets/d/1pqge1MfikjMbSoZXj74ynXllP-HIyx8ofoqRm6g0vAM/edit\#gid\=0, plus other content ids that had legacy_taxons associated with them in publishing api, and for each of them makes a call to remove legacy_taxons from their links if it is present.

[Trello ](https://trello.com/c/gzRFfeSE/1449-write-a-rake-task-in-content-tagger-to-remove-the-legacytaxon-field-from-the-links-payload-of-taxon-content-items-m)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Review by hannako — APPROVED** [review]
LGTM! Thanks Becka 🧹 🪄

File: spec/lib/tasks/taxonomy/remove_legacy_taxon_links_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    content_ids_with_legacy_taxons.each do |content_id|
+      expect(Services.publishing_api).to receive(:patch_links).with(content_id, links: { legacy_taxons: [] }).once
+    end
+
+    content_ids_without_legacy_taxons.each do |content_id|
+      expect(Services.publishing_api).not_to receive(:patch_links).with(content_id, links: { legacy_taxons: [] })
+    end
+
+    expect { rake("taxonomy:remove_legacy_taxon_links") }.to output.to_stdout
```
Comment (sivsubra-tw) [inline spec/lib/tasks/taxonomy/remove_legacy_taxon_links_spec.rb:36]: Is it worth making this slightly more specific e.g. check that the count is present in the output?

Comment (sivsubra-tw) [reply spec/lib/tasks/taxonomy/remove_legacy_taxon_links_spec.rb:36]: Looks good otherwise - thank you!

**Review by sivsubra-tw — APPROVED** [review]

File: spec/lib/tasks/taxonomy/remove_legacy_taxon_links_spec.rb (hunk 1 above)
Comment (BeckaL) [reply spec/lib/tasks/taxonomy/remove_legacy_taxon_links_spec.rb]: Thanks for review ⭐ ! Have updated per your suggestion

**Pull request merged**